test/extended/cluster/context.go

```diff
@@ -17,12 +17,19 @@ const (
 type ContextType struct {
 	ClusterLoader struct {
 		Cleanup    bool
+		Threads    int
 		Projects   []ClusterLoaderType
 		Sync       SyncObjectType  `yaml:",omitempty"`
 		TuningSets []TuningSetType `yaml:",omitempty"`
 	}
 }
 
+type ProjectMeta struct {
```

Comment:
If the intent is to expose these as flags through an `openshift-tests` command, this seems weird.

Reply:
Initially I was just passing `ClusterLoaderType` so the `clWorker` thread would know what to create, but then I created a custom type to wrap some more fields I needed.